types/block.go

```diff
@@ -15,7 +15,7 @@ import (
 
 const (
 	// MaxHeaderBytes is a maximum header size (including amino overhead).
-	MaxHeaderBytes = 511
+	MaxHeaderBytes = int64(511)
```

Comment:
Would `MaxHeaderBytes int64 = 511` be better for these?

Reply:
👍